src/utils.py

```
@@ -21,3 +24,101 @@ async def fetch_channel_messages(
+                )
+                return
+            try:
+                target = await channel.fetch_message(_src_msg.id)
+                if target.content == _src_msg.content:
+                    await target.delete()
+                    try:
+                        _btn.disabled = True
+                        if interaction.message:
+                            await interaction.message.edit(view=view)  # type: ignore[arg-type]
+                    except Exception:
+                        pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)